annotation-processor/src/test/java/org/hibernate/validator/ap/testmodel/annotationparameters/InvalidDigitsParameters.java

```diff
@@ -23,7 +23,27 @@
 	@Digits( integer = -5, fraction = -3 )
 	private BigDecimal decimal3;
 
-	@Digits.List({ @Digits(integer = -5, fraction = -3), @Digits(integer = 5, fraction = -3) })
+	@Digits.List( {@Digits( integer = -5, fraction = -3 ), @Digits( integer = 5, fraction = -3 )} )
 	private BigDecimal decimal4;
 
+	public InvalidDigitsParameters(
+			@Digits( integer = -1, fraction = 3 ) BigDecimal decimal1,
```

Comment:
Same here, it should be: `@Digits(integer = -1, fraction = 3)`.

I would have fixed it when merging but as you are a regular contributor, we might as well have your formatters right ;). We provide IDE configuration here: https://github.com/hibernate/hibernate-ide-codestyles . I know for sure the Eclipse ones are OK but if you use Intellij and there are problems with the configuration file we provide, we might as well fix them! Thanks for your feedback on this!